container/docker/docker.go

```diff
@@ -0,0 +1,144 @@
+package docker
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+	"github.com/wtsi-ssg/wr/container"
+	"github.com/wtsi-ssg/wr/math/conversion"
+)
+
+type Interactor struct {
```

Comment:
Document this. Is your linting turning on and are you running `make lint` before commit/push?

Reply:
Done

Reply:
"Interactor implements the container/Interactor interface for docker."

Reply:
Updated